basculehttp/log.go

```diff
@@ -85,7 +85,7 @@ func SetLogger(logger *zap.Logger) alice.Constructor {
 				source = host
 			}
 
-			logger = logger.With(
+			l := logger.With(
 				zap.Reflect("request.Headers", sanitizeHeaders(r.Header)), //lgtm [go/clear-text-logging]
 				zap.String("request.URL", r.URL.EscapedPath()),
 				zap.String("request.method", r.Method),
```

Comment:
Picking nits:  this should probably be "request.Method" to match the casing of the other fields.